### DISCUSSION TITLE
posts/vim-commands-for-navigation-between-brackets/

### DISCUSSION THREAD

**Discussion by giscus** [m4xshen/m4xshen.dev#10] [Announcements]
# posts/vim-commands-for-navigation-between-brackets/

Discover three powerful Vim commands for blazingly fast navigation between brackets in your code. Enhance your coding efficiency with %, [(, [{, ][, and [] for unmatched precision in large files and complex structures. Perfect for developers looking to streamline their workflow and boost productivity.

https://m4xshen.dev/posts/vim-commands-for-navigation-between-brackets/

**Comment by olivertzeng** [comment]
感謝大大寫這貼文 我真的很喜歡
但是像是python這種沒有分號的程式要怎麼樣跳啊 還是有沒有plugin 可以自動找到/偵測python的indent在冒號時按%也我可以向上面所述跳到迴圈的結尾？

**Reply by m4xshen** [reply]
在Python檔的話我目前沒想到跳到迴圈結尾的方法，但是如果在function內可以用 `[m` 跟 `]M` 跳到function的頭跟尾，我之後再補充到文章內！

**Comment by olivertzeng** [comment]
感謝你我真的很喜歡你的plugins特別是hardtime 雖然讓我有一點hardtime 但是讓我受益無窮！希望將來可以將hardtime 中文化就是了

**Comment by raffaele-lungarella** [comment]
Love it
